projects/core/src/store-finder/facade/store-data.service.spec.ts

```diff
@@ -2,6 +2,13 @@ import { TestBed } from '@angular/core/testing';
 import { PointOfService } from '../../model/point-of-service.model';
 import { StoreDataService } from './store-data.service';
 
+const YEAR_2018 = 2018;
+const MONTH_AUGUST = 8;
+const DAY_SEVENTEEN = 17;
+const DAY_TWENTY_THREE = 23;
+const TO_BE_LATITUDE = 35.528984;
+const TO_BE_LONGITUDE = 139.700168;
```

Comment:
```suggestion
const LATITUDE = 35.528984;
const LONGITUDE = 139.700168;
```
and you can replace values of location.geoPoint with them

Reply:
done